src/app/components/Tables/Reports/Ticket/TicketReportTable.tsx

```diff
@@ -0,0 +1,267 @@
+                      <th className='text-center fs-4 text-white'>ID Tiquete</th>
+                      <th className='text-center fs-4 text-white'>Vendedor</th>
+                      <th className='text-center fs-4 text-white'>Moneda</th>
+                      <th className='text-center fs-4 text-white'>Total</th>
+                      <th className='text-center fs-4 text-white'>Acciones</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {ticketReportPaginated.ticketsCount > 0 &&
+                      ticketReportPaginated.tickets.map((ticket, index) => (
+                        <tr
+                          className='fw-bold fs-6 text-gray-800'
+                          key={`${ticket.ticketCreatedAt}-${index}`}
+                        >
+                          <td className='text-center'>
+                            {DateTime.fromISO(ticket.ticketCreatedAt).toFormat('yyyy-MM-dd')}
+                          </td>
+                          <td className='text-center'>{ticket.ticketId}</td>
+                          <td className='text-center'>{ticket.ticketSoldByUserId}</td>
+                          <td className='text-center'>{currencyCode}</td>
+                          <td className='text-center'>
+                            {formatCurrency(ticket.ticketTotal, currencyCode)}
+                          </td>
+                          <td className='text-center'>
+                            <div
+                              onClick={() => setTicketId(ticket.ticketId)}
+                              style={{
+                                cursor:
+                                  isTicketDetailLoading && ticket.ticketId === ticketId
+                                    ? 'not-allowed'
+                                    : 'pointer',
+                              }}
+                            >
+                              {isTicketDetailLoading ? (
+                                <RenderLoader
+                                  key={ticket.ticketNumber}
+                                  show={isTicketDetailLoading && ticket.ticketId === ticketId}
+                                />
+                              ) : (
+                                <i className='bi bi-zoom-in text-primary fs-2x'></i>
+                              )}
+                            </div>
+                          </td>
+                        </tr>
+                      ))}
+                    {ticketReportPaginated.tickets.length === 0 && (
+                      <tr>
+                        <td colSpan={7} className='text-center'>
+                          No results...
+                        </td>
+                      </tr>
+                    )}
+                  </tbody>
+                </table>
+              </div>
+            </>
+          )}
+        {!isLoading && ticketReportPaginated && ticketReportPaginated.ticketsCount > 0 && (
```

Comment:
when you have multiples conditions to render something, create a variable with a describable name